app/scripts/actions/projectActions.js

```diff
@@ -537,8 +539,8 @@ ProjectActions.editProject.preEmit = function (id, name, desc) {
     });
 };
 
-ProjectActions.loadFolderChildren.preEmit = function (id) {
-    fetch(urlGen.routes.baseUrl + urlGen.routes.apiPrefix + 'folders/' + id + '/children', {
+ProjectActions.getChildren.preEmit = function (id, path) {
```

Comment:
Are there more paths than 'folders/' and 'projects/' ? You might consider making this parameter something like an enumeration i.e. Path.FOLDER or Path.PROJECT so that you're not pushing the URL building _details_ out to the emitters of the action. i.e. if the API changes to 'folder/' you'll need to go find everywhere this is used and update it. By contrast, if you localize that logic into this method and have an abstraction that users of this method can semantically understand, it'll allow you to more easily change the implementation later.